app/translations/en.json

```diff
@@ -1,15 +1,18 @@
+      "installation_instructions": "You can try out Tray.io by [signing up for a free trial](https://tray.io/lp/zendesk/trial?utm_medium=partner&utm_source=directory&utm_campaign=zendesk) or you can [see a personalized demo](https://tray.io/lp/zendesk/demo?utm_medium=partner&utm_source=directory&utm_campaign=zendesk). \n\nWe also recommend reviewing the [Tray.io documentation](https://tray.io/docs), including the [Zendesk Help Article](https://tray.io/docs/article/zendesk)."
+    },
+    "long_description": "Tray believes that any business professional should be able to easily connect together their entire cloud software stack and automate complex processes in order to scale their growth. \n\nCitizen automators in marketing, sales, support and business operations love Tray’s unique solution.\n\n* Connect any web application with Tray’s unique Universal Connector.\n* Get started in minutes with pre-built automations for many popular workflows. \n* Easy drag-and-drop interface that doesn’t require dev resources.\n* Go beyond simplistic point-to-point integrations with our flexible conditional logic operators that handle complex multi-step, multi-app workflows.\n* No limits. No latency. Powerful processing of billions of tasks in real-time with the power of elastic serverless architecture. \n* Prompt and helpful support to ensure your success.\n\nTray.io is a cloud data integration and automation platform built for business people at enterprises and fast-growing companies. Try it out by [signing up for a free trial](https://tray.io/lp/zendesk/trial?utm_medium=partner&utm_source=directory&utm_campaign=zendesk) or [see a personalized demo](https://tray.io/lp/zendesk/demo?utm_medium=partner&utm_source=directory&utm_campaign=zendesk). ",
```

Comment:
Not sure why the long_description and installation_instructions are outside of the blob? Did the compiler fail without placing them there? Please see: https://developer.zendesk.com/apps/docs/developer-guide/multiproduct_apps

Reply:
Good spot